API/src/main/proto/webcast.proto

```diff
@@ -574,41 +574,96 @@ message WebcastLinkMicBattle {
   Common common = 1;
   uint64 id = 2;
   LinkMicBattleConfig battleConfig = 3;
-  uint32 data2 = 4;
+  int32 battleStatus = 4; // SHOULD BE AN ENUM
```

Comment:
You already knows 2 statues, so please create Enum BattleStatus instead of keep using int32. Trust me this will not crash

Reply:
Fixed!